src/main/java/lotto/domain/LottoNumbers.java

```diff
@@ -0,0 +1,62 @@
+package lotto.domain;
+
+import lotto.function.GenerateNumbers;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class LottoNumbers {
+  private static final int LOTTO_NUMBER_TOTAL_COUNT = 6;
+  private static final String EXCEPTION_MESSAGE = "로또는 6자리 숫자로 생성해야 됩니다.";
```

Comment:
```suggestion
  private static final int LOTTO_NUMBER_TOTAL_COUNT = 6;
  private static final String EXCEPTION_MESSAGE = "로또는 " + LOTTO_NUMBER_TOTAL_COUNT + "자리 숫자로 생성해야 됩니다.";
```
EXCEPTION_MESSAGE 또한 다음과 같이 바꿔 볼 수 있겠어요! 😄